README.md

```diff
@@ -1,14 +1,25 @@
 ### Mock APNs Server
 
-Mock APNs / testing server. Supports ssl connections, feedback service, and error responses.
+Mock APNs / testing server supporting ssl connections.
 
-#### Usage
-
-```
-./apnsmock --help
+## Setup
+Before you run the mock server you need to create a ssl certificate
+```bash
+openssl req -new -x509 -keyout key -out cert -nodes
```

Comment:
OpenSSL, by default, uses the PEM format. By convention, the files should have a `.pem` extension (while DER files should have .crt extension)